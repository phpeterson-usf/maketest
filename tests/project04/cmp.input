-h $(repo_dir)/cmp.hex $(TESTS_DIR)/cmp.s
