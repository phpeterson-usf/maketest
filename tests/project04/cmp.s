main:
    cmp r2, r3
    bx lr
