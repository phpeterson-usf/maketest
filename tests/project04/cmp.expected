v2.0 raw
e1520003
e12fff1e
